packages/storybook/src/nimble/tabs/tabs-matrix.stories.ts

```diff
@@ -39,10 +65,15 @@ const component = (
 
 ): ViewTemplate => html`
     <${tabsTag} style="padding: 15px;${widthValue ? ` width: ${widthValue};` : ''}">
```

Comment:
Can we add text labels that describe the cases? There are enough permutations that it's hard to follow without some labels